rlutil.h

```diff
@@ -440,9 +439,7 @@ RLUTIL_INLINE void locate(int x, int y) {
 	SetConsoleCursorPosition(GetStdHandle(STD_OUTPUT_HANDLE), coord);
 #else // _WIN32 || USE_ANSI
 	#ifdef __cplusplus
-		std::ostringstream oss;
-		oss << "\033[" << y << ";" << x << "H";
-		RLUTIL_PRINT(oss.str());
+		RLUTIL_PRINT("\033[" << y << ";" << x << "H");
```

Comment:
> 	inline void RLUTIL_PRINT(RLUTIL_STRING_T st) { std::cout << st; }

Looks like an inline function to me. Perhaps you forgot to commit/push something? `test.cpp` fails to compile here with your branch producing this:

> rlutil.h: In function ‘void rlutil::locate(int, int)’:
rlutil.h:442:27: error: invalid operands of types ‘const char [3]’ and ‘int’ to binary ‘operator<<’
   RLUTIL_PRINT("\033[" << y << ";" << x << "H");